src/content/self-hosted/install/releases.mdx

```diff
@@ -5,7 +5,46 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+
+## 1.4.0
+<TODO: insert release date here once it's published>
+
+### Features
```

Comment:
is kubernetes 1.24 supported as part of this release?

Reply:
Not yet. We'd like to have more certainty before publishing it